shared/src/business/utilities/pdfGenerator/documentTemplates/NoticeOfTrialIssued.test.js

```diff
@@ -0,0 +1,103 @@
+const React = require('react');
+const { mount, shallow } = require('enzyme');
+const { NoticeOfTrialIssued } = require('./NoticeOfTrialIssued.jsx');
+
+describe('NoticeOfTrialIssued', () => {
+  const caseCaptionExtension = 'Petitioner(s)';
+  const caseTitle = 'Test Petitioner';
+  const docketNumberWithSuffix = '123-19S';
+  const trialInfo = {
+    address1: '123 Some St.',
+    city: 'Somecity',
+    judge: 'Judge Dredd',
+    postalCode: '80008',
+    startDate: '02/02/2020',
+    startTime: '9:00 AM',
+    state: 'ZZ',
+  };
+
+  it('renders a document header with trial information', () => {
+    const wrapper = mount(
+      <NoticeOfTrialIssued
```

Comment:
This is reused in all the tests here - could it be pulled up into the describe block?

Reply:
They're almost the same, but not quite. Some use different props and functions.